tests/unit_tests/udf_wrapper_params/filling_mask/single_bfsconn_multiple_subdir_single_model_single_batch.py

```diff
@@ -0,0 +1,60 @@
+from pathlib import PurePosixPath
+from exasol_udf_mock_python.connection import Connection
+from tests.unit_tests.udf_wrapper_params.filling_mask.mock_filling_mask import \
+    MockFillingMaskFactory, MockFillingMaskModel, MockPipeline
+
+
+def udf_wrapper():
+    from exasol_udf_mock_python.udf_context import UDFContext
+    from exasol_transformers_extension.udfs.models.filling_mask_udf import \
+        FillingMask
+    from tests.unit_tests.udf_wrapper_params.filling_mask.mock_sequence_tokenizer \
+        import MockSequenceTokenizer
+    from tests.unit_tests.udf_wrapper_params.filling_mask.\
+        single_bfsconn_multiple_subdir_single_model_single_batch import \
+        SingleBucketFSConnMultipleSubdirSingleModelNameSingleBatch as params
+
+    udf = FillingMask(
+        exa,
+        batch_size=params.batch_size,
+        pipeline=params.mock_pipeline,
+        base_model=params.mock_factory,
+        tokenizer=MockSequenceTokenizer)
+
+    def run(ctx: UDFContext):
+        udf.run(ctx)
+
+
+class SingleBucketFSConnMultipleSubdirSingleModelNameSingleBatch:
```

Comment:
You have dedicated took tests, so I would use here the same k, such that they differ